### DISCUSSION TITLE
Trying to upgrade from 6.2.1

### DISCUSSION THREAD

**Discussion by tommyzzj** [MvvmCross/MvvmCross#4235] [Q&A] [answered]
Hello,

I'm trying to upgrade the MvvmCross from 6.2.1 to target the Android 11 (R API Level 30)

Which version is better between 7.1.2 and 8.0.2 for Xamarin.Droid project?  
Besides, does the MvvmCross 8 support the API Level 31?  I can't any documentation for the api level support

**Comment by Cheesebaron** [comment, marked as answer]
We don't use any API 31 stuff (yet), but using MvvmCross with that API level should not be an issue.

We use AndroidX for fragments and activities, so I don't see any issue with that API level.

**Reply by tommyzzj** [reply]
Hi Cheesebaron, Thanks for the reply. Since I'm targeting API 30 for current AppCompat solution.  Is that fine to go straight forward to the version 8 or 7 would meet this and be more stable?
